src/flutclient.rs

```diff
@@ -140,7 +151,15 @@ where
             }
             Color::W8(white) => u32::from_be_bytes([*white, *white, *white, 0xff]),
         };
-        set_pixel_rgba(self.grids.as_ref(), canvas, x, y, c);
+        set_pixel_rgba(
+            self.grids.as_ref(),
+            canvas,
+            x,
+            y,
+            c,
+            #[cfg(feature = "auth")]
```

Comment:
this is annoying but needed to have the tests work with `--features auth` as well